vmdb/app/models/miq_provision_virt_workflow.rb

```diff
@@ -736,19 +738,14 @@ def allowed_snapshots(_options = {})
   end
 
   def allowed_datastore_storage_controller(_options = {})
-    # TODO: NetAppFiler is an ActAsArModel and doesn't support arel syntax yet
-    NetAppFiler.all.collect(&:name).index_by { |n| n }
+    []
   end
+  deprecate :allowed_datastore_storage_controller
```

Comment:
@tenderlove My commit update stepped on this question from @jrafanie.  I think this is the last issue to be addressed before merging.